appData/src/gb/src/ScriptRunner.c

```diff
@@ -24,7 +24,7 @@ UWORD script_start_stack[STACK_SIZE] = {0};
 
```

Comment:
Took a while, but Menus are broken because max commands is still 6
Line 16, should be `UBYTE script_cmd_args[7] = {0};`
@chrismaltby